help.html

```diff
@@ -23,15 +23,6 @@
     <script src="static/js/jquery-2.1.4.min.js"></script>
     <script src="python-main.js"></script>
 
-    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
-    <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
```

Comment:
This file was for IE8, but it's not even in this repository.
The one below is the warn-user-about-ie-emulation hack from the bootstrap docs.